mimecast/help.md

```diff
@@ -1,5 +1,32 @@
 # Mimecast
 
+## Help
```

Comment:
The Mimecast `help.md` was updated during the hackathon - you'll want to copy/paste your _additions_ to it (the new content as a result of adding a new action) to the existing one

Reply:
Let me know if you need help with this and we can setup a call tomorrow to go over it 👍

Reply:
@mrinehart-r7  It is just a matter of getting the helper_foss repo help.md and plugin.spec.yam for my plugin?